introduction.md

```diff
@@ -6,3 +6,6 @@ My fovorites code school paths are:
 * Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?